src/ThreadedSensitivities.jl

```diff
@@ -62,6 +62,16 @@ function threadedsensitivities(react, paramindices; odesolver=nothing, senssolve
         senssolver = react.recommendedsolver
     end
 
+    # Convert Dict{Any,Any} to Dict{Symbol, Float64}
+    # Needed for pyrms compatability
+    # Python string gets converted to String and runs into expected Symbol, got a value of type String error
+    if odekwargs isa Dict{Any,Any}
```

Comment:
1) I don't think these kwargs are all symbols mapped to floats so if someone tries to turn on an option that isn't a float I think this will crash
2) Can we instead do this the julian way and write a Dict{Any,Any} specific method that just converts the dictionary to Dict{Symbol,Any} and then calls the main method (which I think would need made specific to Dict{Symbol,T})?

Reply:
Good call, I fixed it. I remove the Float64() and the type of the new dictionary should be inferred automatically.

Reply:
I tried using `convert` to convert Dict{Any,Any} to Dict{Symbol,Any}, but it doesn't work.

```
ERROR: MethodError: Cannot `convert` an object of type String to an object of type Symbol
Closest candidates are:
  convert(::Type{T}, ::T) where T at Base.jl:61
  Symbol(::String) at boot.jl:489
  Symbol(::AbstractString) at strings/basic.jl:228
```

Reply:
Oh wait I misunderstood what you meant. I will fix it now